explore/src/main/scala/explore/config/ConfigurationPanel.scala

```diff
@@ -81,16 +80,18 @@ object ConfigurationPanel {
         val showAdvancedCB: Option[Callback] =
           optModeView.get.map(_ => showAdvanced.set(true))
 
+        val posAngleView: View[Option[PosAngleConstraint]] =
+          props.scienceData.undoableView(ObservationData.posAngleConstraint)
```

Comment:
I followed @rpiaggio recommendation about `undoableView` and seems to work quite well